pynvim/msgpack_rpc/event_loop/asyncio.py

```diff
@@ -68,7 +68,7 @@ def pipe_connection_lost(self, fd, exc):
     def pipe_data_received(self, fd, data):
         """Used to signal `asyncio.SubprocessProtocol` of incoming data."""
         if fd == 2:  # stderr fd number
-            self._on_stderr(data)
+            self._on_error(data)
```

Comment:
`_on_error` is for a complete error message, not for streaming data. One could collect all the pieces and add the joined text in the `process_exited` call possibly. Or just change it to `pass` for now.